cron/daily/41-suspend-accounts-without-minimum-amount-of-admins.ts

```diff
@@ -0,0 +1,50 @@
+import '../../server/env';
+
+import { roles } from '../../server/constants';
+import FEATURE from '../../server/constants/feature';
+import POLICIES from '../../server/constants/policies';
+import logger from '../../server/lib/logger';
+import { getPolicy } from '../../server/lib/policies';
+import models, { sequelize } from '../../server/models';
+
+const run = async () => {
+  const collectives = await models.Collective.findAll({
+    include: [
```

Comment:
Shouldn't we pre-filter by `type`? Events and projects inherit the admins, so it doesn't seem necessary to check them.